spec/multisite/flags_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+RSpec.describe "Custom flags in multisite", type: :multisite do
+  describe "#all_flags" do
```

Comment:
```suggestion
  describe "PostACtionType#all_flags" do
```

This makes it clearer which instance method of which class we are testing.